### PR TITLE
WV-3013: Fix dois and short names for AMSR-E and AMSR2 NSIDC layers

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes #WV-3013.

For AMSR-E and AMSR2 layer descriptions:
- Added shortname for AE_Rain for AMSR-E Columnar Water Vapor (day and Night); Surface Precipitation Rate (Day and Night)
- Added shortname and doi for AMSR2 standard products. AU_OCEAN for Cloud Liquid Water (day and night), Total Precipitable Water (Day and Night), Wind Speed (Day and Night); AU_RAIN for Surface Precipitation (Day and Night), Columnar Water Vapor (Day and Night)
- Fixed typos for MODIS/Terra Ice Surface Temperature (Day) and MODIS/Aqua Snow Cover

## How To Test

1. `git checkout wv-3013-nsidc-doi-updates`
2. `npm run build && npm start`
3. Compare [prod link](https://worldview.earthdata.nasa.gov/?l=Coastlines_15m,AMSRE_Surface_Precipitation_Rate_Night,AMSRE_Surface_Precipitation_Rate_Day,AMSRE_Columnar_Water_Vapor_Night,AMSRE_Columnar_Water_Vapor_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2010-01-12-T17%3A40%3A19Z) to [localhost link](http://localhost:3000/?l=Coastlines_15m,AMSRE_Surface_Precipitation_Rate_Night,AMSRE_Surface_Precipitation_Rate_Day,AMSRE_Columnar_Water_Vapor_Night,AMSRE_Columnar_Water_Vapor_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2010-01-12-T17%3A40%3A19Z) to see that AE_Rain has been added at the bottom of the layer description
4. Compare [prod link](https://worldview.earthdata.nasa.gov/?v=-144.7881897540506,-38.623127822541576,32.554658872792196,48.65392930598618&l=Coastlines_15m,AMSRU2_Wind_Speed_Night,AMSRU2_Wind_Speed_Day,AMSRU2_Total_Precipitable_Water_Night,AMSRU2_Total_Precipitable_Water_Day,AMSRU2_Cloud_Liquid_Water_Night,AMSRU2_Cloud_Liquid_Water_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T17%3A40%3A19Z) to [localhost link](http://localhost:3000/?v=-144.7881897540506,-38.623127822541576,32.554658872792196,48.65392930598618&l=Coastlines_15m,AMSRU2_Wind_Speed_Night,AMSRU2_Wind_Speed_Day,AMSRU2_Total_Precipitable_Water_Night,AMSRU2_Total_Precipitable_Water_Day,AMSRU2_Cloud_Liquid_Water_Night,AMSRU2_Cloud_Liquid_Water_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T17%3A40%3A19Z) to check if AU_OCEAN and relevant doi has been added to the bottom of the layer description
5. Compare [prod link](https://worldview.earthdata.nasa.gov/?l=Coastlines_15m,AMSRU2_Surface_Precipitation_Night,AMSRU2_Surface_Precipitation_Day,AMSRU2_Columnar_Water_Vapor_Night,AMSRU2_Columnar_Water_Vapor_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T17%3A40%3A19Z) to [localhost link](http://localhost:3000/?l=Coastlines_15m,AMSRU2_Surface_Precipitation_Night,AMSRU2_Surface_Precipitation_Day,AMSRU2_Columnar_Water_Vapor_Night,AMSRU2_Columnar_Water_Vapor_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T17%3A40%3A19Z) to check if AU_RAIN and relevant doi has been added to the bottom of the layer description
6. Check [MODIS/Terra Ice Surface Temperature Day layer description](http://localhost:3000/?l=Coastlines_15m,MODIS_Terra_Ice_Surface_Temp_Day,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T18%3A01%3A19Z) and see that the extra 1 has been removed "DOI text says "doi:110.5067/MODIS/MOD29P1D.061" which has an extra "1" at the start of the DOI listing."
7. Check [MODIS/Aqua Snow Cover layer description](http://localhost:3000/?l=MODIS_Aqua_NDSI_Snow_Cover,Coastlines_15m,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-01-12-T18%3A01%3A19Z) doi link points to doi:10.5067/MODIS/MYD10_L2.061

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview

### CODE REVIEW

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Day.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ The AMSR2 instrument is a conically scanning passive microwave radiometer. This
 
 The imagery resolution is 2 km and sensor resolution is 6.25 km. The temporal resolution is daily.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP5)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Day.md:7]: This DOI link is NOT FOUND.

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Night.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ The AMSR2 instrument is a conically scanning passive microwave radiometer. This
 
 The imagery resolution is 2 km and sensor resolution is 6.25 km. The temporal resolution is daily.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
\ No newline at end of file
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Night.md:7]: This DOI link is NOT FOUND.

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Total_Precipitable_Water_Day.md [hunk 3]
```diff
@@ -2,4 +2,4 @@ The Total Precipitable Water (Day) layer displays precipitable water totals over
 
 The Advanced Microwave Scanning Radiometer 2 (AMSR2) instrument on the Global Change Observation Mission - Water 1 (GCOM-W1) provides global passive microwave measurements of terrestrial, oceanic, and atmospheric parameters for the investigation of global water and energy cycles. The GCOM-W1 NRT AMSR2 Unified Global Swath Surface Precipitation GSFC Profiling Algorithm is a swath product containing global rain rate and type, calculated by the GPROF 2017 V2R rainfall retrieval algorithm using resampled NRT Level-1R data provided by JAXA. This is the same algorithm that generates the corresponding standard science products in the AMSR SIPS.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
\ No newline at end of file
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Total_Precipitable_Water_Day.md:5]: This DOI link is NOT FOUND.

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Total_Precipitable_Water_Night.md [hunk 4]
```diff
@@ -2,4 +2,4 @@ The Total Precipitable Water (Night) layer displays precipitable water totals ov
 
 The Advanced Microwave Scanning Radiometer 2 (AMSR2) instrument on the Global Change Observation Mission - Water 1 (GCOM-W1) provides global passive microwave measurements of terrestrial, oceanic, and atmospheric parameters for the investigation of global water and energy cycles. The GCOM-W1 NRT AMSR2 Unified Global Swath Surface Precipitation GSFC Profiling Algorithm is a swath product containing global rain rate and type, calculated by the GPROF 2017 V2R rainfall retrieval algorithm using resampled NRT Level-1R data provided by JAXA. This is the same algorithm that generates the corresponding standard science products in the AMSR SIPS.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
\ No newline at end of file
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Total_Precipitable_Water_Night.md:5]: This DOI link is NOT FOUND.

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Wind_Speed_Day.md [hunk 5]
```diff
@@ -2,4 +2,4 @@ The Wind Speed (Day) layer shows wind speed over oceans in meters per second (m/
 
 The Advanced Microwave Scanning Radiometer 2 (AMSR2) instrument on the Global Change Observation Mission - Water 1 (GCOM-W1) provides global passive microwave measurements of terrestrial, oceanic, and atmospheric parameters for the investigation of global water and energy cycles. The GCOM-W1 NRT AMSR2 Unified Global Swath Surface Precipitation GSFC Profiling Algorithm is a swath product containing global rain rate and type, calculated by the GPROF 2017 V2R rainfall retrieval algorithm using resampled NRT Level-1R data provided by JAXA. This is the same algorithm that generates the corresponding standard science products in the AMSR SIPS.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
\ No newline at end of file
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Wind_Speed_Day.md:5]: This DOI link is NOT FOUND.

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Wind_Speed_Night.md [hunk 6]
```diff
@@ -2,4 +2,4 @@ The Wind Speed (Night) layer shows wind speed over oceans in meters per second (
 
 The Advanced Microwave Scanning Radiometer 2 (AMSR2) instrument on the Global Change Observation Mission - Water 1 (GCOM-W1) provides global passive microwave measurements of terrestrial, oceanic, and atmospheric parameters for the investigation of global water and energy cycles. The GCOM-W1 NRT AMSR2 Unified Global Swath Surface Precipitation GSFC Profiling Algorithm is a swath product containing global rain rate and type, calculated by the GPROF 2017 V2R rainfall retrieval algorithm using resampled NRT Level-1R data provided by JAXA. This is the same algorithm that generates the corresponding standard science products in the AMSR SIPS.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
\ No newline at end of file
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP)
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/amsr2/AMSRU2_Wind_Speed_Night.md:5]: This DOI link is NOT FOUND.

**Review by Tomcariello — CHANGES_REQUESTED** [review]

File: config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Day.md (hunk 1 above)
Comment (minniewong) [reply config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Day.md]: Looks like they gave me the wrong link and was a number off, I've fixed them!

**Review by ryanweiler92 — APPROVED** [review]

**Review by Tomcariello — APPROVED** [review]

**Pull request merged**